lib/bolt/bolt_option_parser.rb

```diff
@@ -7,6 +7,7 @@
 module Bolt
   class BoltOptionParser < OptionParser
     OPTIONS = { inventory: %w[nodes targets query rerun description],
+                group: %w[nodes targets query rerun description],
```

Comment:
I don't think this command will actually do anything with these options. I'd remove this, and modify the line below to not include it.